app/templates/components/raw-config.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="inner-yaml-container">
-  <div class="file-name">
+  <div class="header" {{action 'toggle'}}>
```

Comment:
I would use the `on` modifier here to make this more explicit, like: `{{on 'click' (action 'toggle')}}`

Here is some more information about it https://github.com/emberjs/rfcs/blob/master/text/0471-on-modifier.md